src/dbnode/persist/fs/write.go

```diff
@@ -40,6 +40,10 @@ import (
 	xtime "github.com/m3db/m3x/time"
 )
 
+const (
+	checkpointFileSizeBytes = 4
```

Comment:
maybe have this and `digest/digestLen` be exported types and ensure they're the same in a test.

Reply:
done